utils/round_trip.rb

```diff
@@ -30,7 +30,7 @@
     generated_adoc = Coradoc::Generator.gen_adoc(doc)
     cleaned_adoc = Coradoc::Input::HTML.cleaner.tidy(generated_adoc)
     File.open("#{file_path}.roundtrip","w"){|f| f.write(cleaned_adoc)}
-    `diff -B #{file_path} #{file_path}.roundtrip > #{file_path}.roundtrip.diff`
+    `diff -BNaur #{file_path} #{file_path}.roundtrip > #{file_path}.roundtrip.diff`
```

Comment:
Better write a separate RSpec matcher that compares 2 Coradoc Document trees?

Reply:
It is just an utility at this point. It would make sense to make some roundtripping tests, but at the time we are not at 100% coverage.